editor/plugins/shader_editor_plugin.h

```diff
@@ -94,13 +94,13 @@ class ShaderEditor : public PanelContainer {
 		BOOKMARK_GOTO_NEXT,
 		BOOKMARK_GOTO_PREV,
 		BOOKMARK_REMOVE_ALL,
-
+		HELP_DOCS,
 	};
 
 	MenuButton *edit_menu;
 	MenuButton *search_menu;
 	MenuButton *bookmarks_menu;
-	MenuButton *settings_menu;
```

Comment:
I really wonder why compilers don't catch such unused variables? It's the second occurrence in a few days, feels like we could do a good cleanup with proper warnings.